libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -3846,6 +3846,38 @@
     <input name="in" type="integer" value="0" />
     <output name="out" type="float" default="0.0" />
   </nodedef>
+  <nodedef name="ND_convert_color3_shader" node="convert" version="1.0" isdefaultversion="true" nodegroup="shader" doc="Convert color3 to shader">
```

Comment:
Perhaps the naming convention could be ND_xxx_surfaceshader, to clarify that output type is "surfaceshader", and avoid conflicts in the future with other shader types?

Reply:
Sounds good. I'll make the change.